src/main/java/com/example/springbootboardjpa/dto/response/PostResponse.java

```diff
@@ -0,0 +1,34 @@
+package com.example.springbootboardjpa.dto.response;
+
+import com.example.springbootboardjpa.entity.Post;
+import com.example.springbootboardjpa.entity.User;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor
+public class PostResponse {
+
+    private Long postId;
+    private String title;
+    private String content;
+    private User userId;
```

Comment:
userId는 Long타입으로 확인되는데 현재 타입이 User로 되어있네요!
혹시 다른 의도하신 바가 있으실까요?!

```suggestion
    private Long userId;
```

Reply:
반영: ef7130219ec87023c2647e7c3b3bebb1771dca47
코드내에서 자료형이 잘못되어 있는지 미처 확인하지 못했습니다! `Long`타입으로 수정하였습니다!